source/crud/tests/README.rst

```diff
@@ -39,22 +39,34 @@ Test Format
 
 Each YAML file has the following keys:
 
+- ``runOn`` (optional): An array of server version and/or topology requirements
+  for which the tests can be run. If the test environment satisfies one or more
+  of these requirements, the tests may be executed; otherwise, this file should
+  be skipped. If this field is omitted, the tests can be assumed to have no
+  particular requirements and should be executed. Each element will have some or
+  all of the following fields:
+
+  - ``minServerVersion`` (optional): The minimum server version (inclusive)
+    required to successfully run the tests. If this field is omitted, it should
+    be assumed that there is no lower bound on the required server version.
+
+  - ``maxServerVersion`` (optional): The maximum server version (inclusive)
+    against which the tests can be run successfully. If this field is omitted,
+    it should be assumed that there is no upper bound on the required server
+    version.
+
+  - ``topology`` (optional): An array of server topologies against which the
+    tests can be run successfully. Valid topologies are "single", "replicaset",
+    and "sharded". If this field is omitted, the default is all topologies (i.e.
+    ``["single", "replicaset", "sharded"]``).
+
```

Comment:
Might have been missed from my previous comment, but can you add a note about `minServerVersion` and `maxServerVersion` under https://github.com/mongodb/specifications/blob/master/source/crud/tests/README.rst#legacy-test-format-for-single-operations? Feel free to copy-paste this:

> Instead of a top-level `runOn` field, server requirements are denoted by separate top-level `minServerVersion` and `maxServerVersion` fields. The minimum server version is an inclusive lower bound for running the test. The maximum server version is an exclusive upper bound for running the test. If a field is not present, it should be assumed that there is no corresponding bound on the required server version.

So long as the v1 directory exists, I'd like for that section to remain in sync and cover any test fields that would not be found in the v2 format.